docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -533,19 +537,6 @@ It cannot be used to compare one pipeline to another or even one process to _its
 A pipeline with only one single-threaded input may contribute up to 1.00, a pipeline whose inputs have hundreds of inbound connections may contribute much higher numbers to this combined value.
 
 Additionally, some amount of back-pressure is both _normal_ and _expected_ for pipelines that are _pulling_ data, as this back-pressure allows them to slow down and pull data at a rate its downstream pipeline can tolerate.
-
```

Comment:
since this section is about process-level flow rates, I have snipped these to a new section under the pipeline responses.